app/controllers/timelineable.rb

```diff
@@ -6,13 +6,13 @@ module Timelineable
   end
 
   def timeline
-    @display_date = display_date_as_time.to_date
+    @display_date = display_date_as_time
```

Comment:
Maybe `@display_date_as_time`? I'd really like to have the word `time` in there somewhere.